exp/operator/config/default/kustomization.yaml

```diff
@@ -1,36 +1,39 @@
 # Adds namespace to all resources.
-namespace: operator-system
+namespace: capi-operator-system
 
 # Value of this field is prepended to the
 # names of all resources, e.g. a deployment named
 # "wordpress" becomes "alices-wordpress".
 # Note that it should also match with the prefix (text before '-') of the namespace
 # field above.
-namePrefix: operator-
+namePrefix: capi-operator-
 
 # Labels to add to all resources and selectors.
-#commonLabels:
-#  someName: someValue
+commonLabels:
+  clusterctl.cluster.x-k8s.io/core: "capi-operator"
```

Comment:
Why is it `clusterctl.cluster.x-k8s.io`? Did we agree on this somewhere? It seems a bit odd to me to have this clusterctl label key in the operator labels.

Reply:
We are already applying clusterctl labels to all the providers, and to cert-manager, I think this makes things consistent and it will make easier for clusterctl to manage the operator if required (e.g for upgrades)